docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.8"
+
+services:
+  web:
+    build: .
+    command: bash -c "python3 manage.py makemigrations &&
+      python3 manage.py migrate &&
+      python3 manage.py runserver 0.0.0.0:8000"
```

Comment:
Could move `makemigrations` and `migrate` out of the top level command and instead in the docs to run separately, so that we have more control when they are run and so that the runserver is run as the top level process.